### ISSUE TITLE
Causing JVM to crash in conjunction with javafx

### ISSUE THREAD

**Issue by JoshuaD84** [kwhat/jnativehook#185]
Full summary here: https://stackoverflow.com/questions/45262517/fatal-error-when-calling-showsavedialog

**Comment by kwhat** [comment]
Hi,

I took a look through the stack trace and it doesn't seem to crash in the library directly.  My only suggestion at this time would be to try version 2.0.  There are some very strange crashes coming out of xcb right now with 2.1 on linux.  I am currently in the process if refactoring that issue and may remove xcb if it is too unstable.  

Please let me know if you have the same issue with the 2.x branch.

**Comment by JoshuaD84** [comment]
JNativeHook 2.0.2 works, thank you!!  I love your library, glad that this issue was able to be avoided.

**Comment by kwhat** [comment]
I am going to leave this open until i figure out what is going on with xcb.

**Comment by kwhat** [comment]
Ok this should address a number of issues that people have been having including this one.  Please test and let me know if this issue can be closed.

[JNativeHook-2.1.1-rc](https://oss.sonatype.org/content/repositories/snapshots/com/1stleg/jnativehook/2.1.SNAPSHOT/jnativehook-2.1.20170730.201417-33.jar)

[JNativeHook-2.0.4-rc](https://oss.sonatype.org/content/repositories/snapshots/com/1stleg/jnativehook/2.0.SNAPSHOT/jnativehook-2.0.20170730.201637-41.jar)

**Comment by JoshuaD84** [comment]
Sorry for taking so long to test. 

That link you sent is now invalid. I went here:
https://oss.sonatype.org/content/repositories/snapshots/com/1stleg/jnativehook/2.1.SNAPSHOT/
and got: jnativehook-2.1.20170819.180800-42.jar

The bug outlined above no longer occurred, although the hotkeys no longer worked. I haven't yet figured out whether this is on my side or the libraries. Will post another bug if there is one, but for now it seems to me that the bug I filed about is resolved.
